eks/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+ 
+CLEAN=${CLEAN:-true}
+
+if [[ $CLEAN == true ]]; then
+    echo -e "\nCleaning up Kubera."
+    echo "Set CLEAN=false if you wish for this not to occur."
+    helm uninstall kubera -n kubera
+    echo "Delete PVC"
+    kubectl delete pvc --all -n kubera
```

Comment:
Logs give error on deleting PVCs. Are these actually deleted?

Reply:
There is one typo in command thatswhy it was giving error. I will fix this in next commit. @AmitKumarDas